util-network/src/main/kotlin/com/tailoredapps/androidutil/network/emptynetworkresponse/EmptyNetworkResponseExt.kt

```diff
@@ -53,6 +53,7 @@ fun Observable<EmptyNetworkResponse>.split(): Observable<Unit> {
  * Extension function that folds a [EmptyNetworkResponse] to a wrapped type that has an success and an error state.
  */
 @CheckReturnValue
+@Deprecated("Renamed function to reflect it's actual functionality", replaceWith = ReplaceWith("this.wrap(successCreator, errorCreator)"))
```

Comment:
Nice usage of the `@Deprecated` annotation with editor actions :+1:

Reply:
👌